jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/insights/TimeUnit.kt

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.cloudwatch.logs.insights
+
+import software.aws.toolkits.resources.message
+import java.time.temporal.ChronoUnit
+
+enum class TimeUnit(val unit: ChronoUnit, val text: String) {
+    MINUTES(ChronoUnit.MINUTES, message("cloudwatch.logs.time_minutes")),
+
```

Comment:
nit: the spacing is a bit weird, why are there line returns between each entry?

Reply:
fixed